Makefile

```diff
@@ -14,8 +14,15 @@ doc: fmt
 
 test: fmt build-test-wasms test-only
 
+# Run tests. 
+# The docs feature is excluded because it is a market for docs builds. The
+# hazmat granular features are excluded because all hazmat features are tested
+# together with the umbrella hazmat feature.
 test-only:
-	cargo hack --feature-powerset --ignore-unknown-features --features testutils --exclude-features docs test
+	cargo hack --feature-powerset --ignore-unknown-features --features testutils \
+		--exclude-features docs \
+		--exclude-features hazmat-crypto \
```

Comment:
Nit: all new hazmat sub types need to be included here in the future? would it be better to exclude 'hazmat' instead (test with sub features combinations), since this is easier to maintain and also more aligned with what cargo hack is intended for?

Reply:
The reason we do this isn't to exclude Soroban, it's to group the features together. See this comment above. 
https://github.com/stellar/rs-soroban-sdk/pull/1639#discussion_r2612583311